contrib/kube-prometheus/hack/grafana-dashboards-configmap-generator/templates/grafana-deployment-original.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: grafana
```

Comment:
I'm talking about this file regarding the "original" deployment, I think this might be here by mistake as I can't see it being used anywhere. I might be wrong though and just can't see where it's used :slightly_smiling_face: .

Reply:
you are right :-) removed!
one important thing to agree: If we follow up this approach, now the main grafana manifest to maintain and keep updated will be `grafana-deployment-template.yaml`.

If anyone has a better way to adapt the grafana-deployment manifest to multiple configmaps  than the `sed` commands I'm using please let me know...

Reply:
I'm perfectly fine with the `sed` approach you took, the only sort of precedence where we're doing "templating" [here](https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus/hack/scripts/generate-grafana-credentials-secret.sh) close to this, but was far simpler. So I think the way you did is just fine.